### DISCUSSION TITLE
Zig bindings for libcurl

### DISCUSSION THREAD

**Discussion by jiacai2050** [curl/curl#14432] [Show and tell]
Hi everyone, 

I create a [Zig binding](https://github.com/jiacai2050/zig-curl) for libcurl last year, both easy and multi API are supported.

The API hasn't got any changes for a while, and there are CI to protect every commit from breaking, a quick GitHub search show it already got adopted by real projects:
- https://github.com/Syndica/sig/blob/cccc96ce067af6103c93807bb461d90b55d9db15/build.zig.zon#L31

So I wonder if I can get it listed at https://curl.se/libcurl/bindings.html ?

**Comment by bagder** [comment]
Absolutely, just make a PR adding it to the [docs/BINDINGS.md](https://github.com/curl/curl/tree/master/docs/BINDINGS.md) file and it will appear on the site automatically.

**Reply by jiacai2050** [reply]
Thanks, working on it!
